### ISSUE TITLE
Exception handling

### ISSUE THREAD

**Issue by quackersian** [quackersian/space-bot#11]
Need to handle exceptions such as
`discord.ext.commands.errors.CommandNotFound: Command "fact" is not found`

**Comment by quackersian** [comment]
Maybe log who used invalid command

**Comment by quackersian** [comment]
possible fix:
```
from discord.ext.commands import CommandNotFound

@bot.event
async def on_command_error(ctx, error):
    if isinstance(error, CommandNotFound):
        return
    raise error
```

**Comment by quackersian** [comment]
Fixed in v0.4.2
